frontend/src/features/admin-form/create/workflow/components/WorkflowContent/InactiveStepBlock/InactiveStepBlock.tsx

```diff
@@ -190,18 +179,19 @@ export const InactiveStepBlock = ({
           ) : null}
         </Stack>
       </chakra.button>
-      {!isFirstStep && (
+      {
         <IconButton
           top={{ base: '0.5rem', md: '2rem' }}
           right={{ base: '0.5rem', md: '2rem' }}
           pos="absolute"
-          aria-label="Delete step"
+          aria-label="Click to edit"
           variant="clear"
-          colorScheme="danger"
-          onClick={handleOpenDeleteModal}
-          icon={<BiTrash fontSize="1.5rem" />}
+          colorScheme="secondary"
```

Comment:
Checked with design, we should be using `primary` instead. We can omit this line since `colorScheme` defaults to `primary` if not defined.